.github/workflows/commit_new_release_to_main.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - name: Merge release into main with priority on main changes
       run: |
         git checkout main
-        git merge --strategy-option=ours release -m "Merge release into main prioritizing main changes"
+        git merge --strategy-option=ours origin/release -m "Merge release into main prioritizing main changes"
```

Comment:
can you please update this code to us `actions/checkout@v3`? All the other actions use that and It would be nice to keep it all consistent.